pkg/store/store.go

```diff
@@ -34,42 +34,45 @@ type Store struct {
 }
 
 var Default = struct {
```

Comment:
I just re-ordered stuff in here to be sorted, so that future merges will be simpler.
I also renamed `ManagedBy` to `LabelValueManagedBy` and added `LabelKeyManagedBy`